src/codegen/abi.js

```diff
@@ -239,6 +239,7 @@ module.exports = class AbiCodeGenerator {
           parentField,
         )
       : {
+          name: [],
```

Comment:
What's this for?

Reply:
Good catch, the `name` property on this result of this function doesn't get used.  

The other conditional arm here resolves to an object with a `name` property; in order to keep the return type of `_generateInputOrOutput` consistent I can delete `name` wherever it would be returned.

So we'd get something along the lines of
```rust
return valueType === 'tuple'
      ? this._generateTupleType(
          inputOrOutput,
          index,
          parentClass,
          parentType,
          parentField,
        ).delete('name')
      : {
          getter: tsCodegen.method(...
```